### PR TITLE
Extract modules from cardano-node to create package cardano-cli

### PR DESCRIPTION
**Pull request by intricate**

### CODE REVIEW

File: cardano-cli/cardano-cli.cabal [hunk 1]
```diff
@@ -0,0 +1,123 @@
+name:                  cardano-cli
+-- TODO @intricate: Should this match the `cardano-node` version?
+version:               0.0.1
```
Comment (intricate) [inline cardano-cli/cardano-cli.cabal:3]: @erikd @Jimbo4350

Comment (erikd) [reply cardano-cli/cardano-cli.cabal:3]: I think all packages in this repo should have the same version number. @dcoutts ?

Comment (intricate) [reply cardano-cli/cardano-cli.cabal]: Set it to the same version as `cardano-node`.

File: cardano-cli/NOTICE [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Copyright 2019 Input Output (Hong Kong) Ltd.
```
Comment (Jimbo4350) [inline cardano-cli/NOTICE:1]: Should be 2020?

Comment (intricate) [reply cardano-cli/NOTICE:1]: Right

**Pull request merged**